src/Worlds.scss

```diff
@@ -176,6 +176,6 @@ body.contrast-theme {
     }
 
     .Scene__path-line--Space {
-        stroke: #FFF500;
```

Comment:
Since it's now the same colour for all worlds, it seems simpler just to have one rule for `.Scene__path-line`

Reply:
With same reasoning as the comment I left in PR-469, this is some duplication we are expecting.